src/EditorFeatures/Core/Implementation/AutomaticCompletion/AbstractAutomaticLineEnderCommandHandler.cs

```diff
@@ -55,12 +52,12 @@ internal abstract class AbstractAutomaticLineEnderCommandHandler :
         /// </summary>
         protected abstract bool TreatAsReturn(Document document, int position, CancellationToken cancellationToken);
 
-        public CommandState GetCommandState(AutomaticLineEnderCommandArgs args, Func<CommandState> nextHandler)
+        public VisualStudio.Commanding.CommandState GetCommandState(AutomaticLineEnderCommandArgs args, Func<VisualStudio.Commanding.CommandState> nextHandler)
         {
-            return CommandState.Available;
+            return VisualStudio.Commanding.CommandState.CommandIsAvailable;
         }
 
-        public void ExecuteCommand(AutomaticLineEnderCommandArgs args, Action nextHandler)
+        public void ExecuteCommand(AutomaticLineEnderCommandArgs args, Action nextHandler, CommandExecutionContext context)
         {
             // get editor operation
             var operations = _editorOperationsFactoryService.GetEditorOperations(args.TextView);
```

Comment:
the next line (that checks of operations is null) seems to eb gone.

Reply:
I can see it here below